R/app.R

```diff
@@ -1,17 +1,7 @@
-#
-# This is a Shiny web application. You can run the application by clicking
-# the 'Run App' button above.
-#
-# Find out more about building applications with Shiny here:
-#
-#    http://shiny.rstudio.com/
-#
-
+# Shiny router test
 library(shiny)
 library(shiny.router)
 
-p404 <- page404(message404 = "Nie ma takiej strony")
-
 menu <- (
   tags$ul(
     tags$li(a(class = "item", href = "/", "Page")),
```

Comment:
Can you try to move `textInput("number", "Numerek"),` outside the page and insert it exactly in `shinyUI` funciton? It will result with not refreshing it when we change the page.